src/features/security-cloud-game/SecurityCloudGame.scss

```diff
@@ -30,10 +30,14 @@
 
     .bdl-SecurityCloudGame-message {
         flex: none;
+        max-height: 40px;
```

Comment:
does the height change now? since the content doesn't wrap anymore would this be a fixed `height` instead of `max-height`?